views/components/Graphs/BarGraph.jsx

```diff
@@ -29,9 +31,23 @@ const BarGraph = ({ graph }) => {
               study !== constants.TOTALS_STUDY &&
               valueLabel !== constants.NOT_AVAILABLE
             return showToolTip
-              ? `Site:${study} \n Current: ${count} \n Target: ${studyTarget}`
+              ? `${study} target: ${
+                  graph.studyTotals[study].targetTotal
+                } (100%) \n ${study} current: ${
+                  graph.studyTotals[study].count
+                } (${toolTipPercent(
+                  graph.studyTotals[study].count,
+                  graph.studyTotals[study].targetTotal
+                )}%)\n ${valueLabel} target: ${studyTarget} (${toolTipPercent(
+                  +studyTarget,
+                  graph.studyTotals[study].targetTotal
+                )}%) \n ${valueLabel} current: ${count} (${toolTipPercent(
+                  count,
+                  graph.studyTotals[study].targetTotal
+                )}%)`
               : null
```

Comment:
This is a bit tough to read, what do you think about breaking the pieces out into named consts and building the final string out of those?
Also I _think_ the white space around some of the `\n` chars above are so things look centered in the tooltip? If so, could we add something to line 50 below in the style object instead to achieve that?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/149/commits/ae1bdc5023821f2b78f966b51b312de7c34449be

Reply:
> the \n char is to make the new line

👍 yep just looking at the white space chars on either side of it. ae1bdc5 looks great!

Reply:
🎱 🎱 could make each "line" it's own variable
up to you!